src/EC2Store.js

```diff
@@ -22,7 +22,59 @@ class EC2Store {
 			let ec2 = new AWS.EC2();
 			ec2.describeSnapshots({}, function (err, response) {
 				if (err) reject(err);
-				else resolve(response);
+
+				else {
+					var taggedForBackup = response.Snapshots.filter(function(obj){
+						for (var i=0; i<obj.Tags.length; i++) {
+							if (obj.Tags[i].Key === 'backups:config-v0') {
+								return true;
+							}
+						}
+						return false;
+					});
+
+					resolve(taggedForBackup.map(function(obj){
+
+						var filteredForName = obj.Tags.filter(function(obj){
+							if (obj.Key === 'Name'){
+								return true;
+							} else {
+								return false;
+							}
+						});
+
+						if (filteredForName.length === 1) {
+							filteredForName = filteredForName[0];
+						} else {
+							throw new Error('expected to receive snapshot with a single value for name but length > 1');
+						}
+
+						var filteredForDate = obj.Tags.filter(function(obj){
+							if (obj.Value.slice(0,10) === 'ExpiryDate'){
+								return true;
+							} else {
+								return false;
+							}
+						});
+
+						if (filteredForDate.length === 1) {
+							filteredForDate = parseInt(filteredForDate[0].Value.slice(11,23));
+						} else {
+							throw new Error('expected to receive snapshot with a single value for expiry date but length > 1');
+						}
+
+						var obj2 = {
+							SnapshotId: obj.SnapshotId,
+							StartTime: obj.StartTime,
+							[filteredForName.Key]: filteredForName.Value,
```

Comment:
Since `[filteredForName.Key]` is supposed to always `Name`, it would be clearer if it just said `Name` here.